api/types.proto

```diff
@@ -277,6 +277,40 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum OnFailure {
+		PAUSE = 0;
+		CONTINUE = 1;
+		// TODO(aaronl): Add ROLLBACK as a supported failure mode.
+		// (#486)
+	}
+
+	// OnFailure is the action to take when an update failures. Currently,
+	// a failure is defined as a single updated task failing to reach the
+	// RUNNING state. In the future, there will be configuration to define
+	// what is treated as a failure (see #486 for a proposal).
+	OnFailure on_failure = 3;
+}
+
+// UpdateStatus is the status of an update in progress.
+message UpdateStatus {
+	enum UpdateState {
```

Comment:
Should we have a `NEW` default for the zero value to ensure that `RUNNING` is a conscious state?